src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionHotlineViewController.swift

```diff
@@ -85,6 +88,16 @@ class ExposureSubmissionHotlineViewController: DynamicTableViewController {
 			]
 		)
 	}
+
+	/// Comment me.
```

Comment:
That was probably a TODO?

Reply:
oops overlooked. thanks for pointing it out!